libs/langchain/langchain/chains/combine_documents/stuff.py

```diff
@@ -95,6 +96,15 @@ def format_docs(inputs: dict) -> str:
     ).with_config(run_name="stuff_documents_chain")
 
 
+@deprecated(
+    since="0.2.13",
+    removal="1.0",
```

Comment:
1.0 or 0.3?

Reply:
It was decided to save 0.3 for pydantic changes and not remove anything in 0.3. After that decision, some deprecations were set for a 0.4+ removal.

In June it was then decided to remove chains in 1.0 (right after that discussion I updated it for LLMChain in https://github.com/langchain-ai/langchain/pull/23373).

I have a separate todo to update all removal targets to 1.0. OK to revisit this decision and keep some in 0.4 if we like.